tools/collaboration_tools.md

```diff
@@ -58,15 +58,15 @@ Typically approved requests include: GitHub, tool requests, or other tools/platf
 
 ## Mailing lists
 
-Academy Software Foundation maintains a number of mailing lists for the foundation itself, its members, and the community at large. These mailing lists are hosted at [groups.io](https://groups.io) and managed by the Linux Foundation staff and Academy Software Foundation community leadership.
+Academy Software Foundation maintains a number of mailing lists for the foundation itself, its members, and the community at large. These mailing lists are managed by the Linux Foundation staff and Academy Software Foundation community leadership.
```

Comment:
Does it make sense to mention that at this point, mailing list traffic has slowed to a trickle, and that there is more likely to be feedback to Slack posts?

Reply:
Both are tools to use, and we do see some traffic depending on the project. It might be good to provide some advise on which medium to use for what perhaps.